docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,311 @@
+# Privacy First: Send Payjoin Transactions with Payjoin Dev Kit
+
+Want to enhance your Bitcoin transactions with enhanced privacy and reduced
+fees? Look no further than Payjoin and the powerful rust-payjoin crate! Payjoin
+allows you to combine your transaction with others, minimizing individual
+footprints and potentially saving on network fees.
+
+This tutorial will guide you through the exciting world of Payjoin using Rust.
+We'll delve into the rust-payjoin library, providing step-by-step instructions
+on crafting and sending your very own Payjoin transactions. Moreover, We'll
+guide you through setting up payjoin-cli, and use it a the receiver side in the
+payjoin transaction.
```

Comment:
Pleasantly surprised by the prose generated here. I think we still want to focus on how it's the `payjoin` crate (since that's how it exists on crates.io) and `send` module.